bioinput.py

```
@@ -1,10 +1,33 @@
-class Bioinput():
-    """
+import pandas as pd
+
+
+class Bioinput:
+    """Input Class
 
     """
 
     def __init__(self, file="ecg_data.csv"):
         self.file = file
+        self.ecg_dataframe = self.read_input()
 
     def read_input(self):
-        print("BME590")
\ No newline at end of file
+        """Read in a CSV file and convert it into a DataFrame with the columns being time and voltage"
+
+        :return: A DataFrame for the time and voltage ecg data
+        """
+
+        try:
+            self.ecg_dataframe = pd.read_csv(self.file, names=['time', 'voltage'], skiprows=1)
+            if len(self.ecg_dataframe) == 0:
+                print("No data found")
```

Is an empty data file an error condition (e.g. would you want to throw an exception here to interrupt and exit the program)?